src/templates/macros.html

```diff
@@ -0,0 +1,59 @@
+{% macro summary_line(kind, covered, total) %}
+    {% set per = percent(num=covered, den=total) %}
```

Comment:
To be clear I don't know `tera` so maybe my next questions are useless.
Isn't overkill to call `percent` function using a `HashMap` ?
Isn't possible to just call `percent(covered, total)` ?
`percent` is just an example here

Reply:
It is not possible, tera uses that way to pass the arguments.

Reply:
Ok. It's kinda stupid... because finally we could just pass the arguments in an array and an array is finally a hashmap with keys equal to 0 & 1 here...